src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/services.po.ts

```diff
@@ -37,7 +37,7 @@ export class ServicesPageHelper extends PageHelper {
     });
   }
 
-  addService(serviceType: string, exist?: boolean, count = '1') {
+  addService(serviceType: string, exist?: boolean, snmpVersion?: string, count = '1') {
```

Comment:
i think this is better to avoid more changes to the already existing tests. Otherwise you would need to adapt [these parts too](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/03-create-cluster-create-services.e2e-spec.ts#L12)
```suggestion
  addService(serviceType: string, exist?: boolean, count = '1', snmpVersion?: string) {
```